src/ql/pass/map/qubits/place_mip/place_mip.cc

```diff
@@ -90,81 +93,77 @@ PlaceQubitsPass::PlaceQubitsPass(
     const utils::Ptr<const pmgr::Factory> &pass_factory,
     const utils::Str &instance_name,
     const utils::Str &type_name
-) : pmgr::pass_types::KernelTransformation(pass_factory, instance_name, type_name) {
+) : pmgr::pass_types::Transformation(pass_factory, instance_name, type_name) {
     options.add_int(
         "horizon",
         "When specified, the placement algorithm will only consider the "
-        "connectivity required to perform the first N two-qubit gates of each "
-        "kernel. When not specified or zero, all two-qubit gates are "
+        "connectivity required to perform the first N most frequent two-qubit gate types. "
+        "When not specified or zero, all two-qubit gate types are "
```

Comment:
What do you mean by gate type?

Reply:
The algorithm takes as input a map, mapping pairs of qubits to number of occurrence of such 2q gate in the input circuit.
What I call a gate type is a pair of qubit indices, which represents a 2q gate.
I haven't found a better designation for this, if you do please lmk.

So with the following circuit:
cnot 2, 5
cz 2, 5
cnot 2, 5

then (2, 5) will map to 3, and (2, 5) is a 2q gate "type"